src/core/qgismobileapp.cpp

```diff
@@ -198,6 +236,8 @@ QgisMobileapp::QgisMobileapp( QgsApplication *app, QObject *parent )
   mOfflineEditing = new QgsOfflineEditing();
 
   mSettings.setValue( "/Map/searchRadiusMM", 5 );
+
+  ( void )new AppMissingGridHandler( this );
```

Comment:
Can we store this pointer somewhere (e.g. in a member variable)?
Sooner or later we'll want to react to missing grids, probably by connecting to signals from this handler.

Reply:
Done.